algorithm: cs_merge_icp_gradient
slam_preprocessing_enabled: true
known_preprocessing_enabled: true
known_map_config_file: data/campus.yaml
known_map_frame_id: /known_map_localization/anchor

# SLAM map unit size [meters]
slam_scale_mode: gps #[parameter/alignment/gps]
slam_map_scale: 9.6

# algorithm parameters are specified in launch file of the cs_merge nodes

# filter parameters
gps_constraint_radius: 2.
aging_rate: 0.95
